tests/test_merger.py

```diff
@@ -117,7 +113,50 @@ def test_merge():
 
     # TODO: There seem to be no destinations for those links?
 
-    # Clean up
+
+tmp_path = "dont_commit_merged.pdf"
+
+
+def test_merger_operations_by_traditional_usage():
+    # Arrange
+    merger = PdfMerger()
+    merger_operate(merger)
+
+    # Act
+    merger.write(tmp_path)
+    merger.close()
+
+    # Assert
+    check_outline(tmp_path)
+
+    # cleanup
+    os.remove(tmp_path)
```

Comment:
Same question here.

Reply:
I don't think so. I guess we always want to delete those. I don't have a suspicion (for this test) that the content being written is wrong.